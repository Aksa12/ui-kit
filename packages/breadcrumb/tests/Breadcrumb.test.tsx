import React from "react";
import { render } from "enzyme";
import toJson from "enzyme-to-json";

import { Breadcrumb } from "../";

describe("Breadcrumb", () => {
  it("default", () => {
    expect(
      toJson(
        render(
          <Breadcrumb>
            <span>One</span>
            <span>Two</span>
          </Breadcrumb>
        )
      )
    ).toMatchSnapshot();
  });
});
