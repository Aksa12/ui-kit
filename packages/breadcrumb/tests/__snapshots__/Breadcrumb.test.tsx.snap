// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Breadcrumb default 1`] = `
<div
  class="css-ewxxaj"
>
  <span>
    One
  </span>
  <span>
    Two
  </span>
</div>
`;
